README.md

```diff
@@ -94,41 +186,82 @@ TrajectoryConverter(input_data).write_JSON("output_file_name")
 ### Add metrics data to plot
 See the [Plots Tutorial](examples/Tutorial_plots.ipynb) for details. An overview:
 ```python
+import numpy as np
 from simulariumio import ScatterPlotData
 
+converter = TrajectoryConverter(input_data) # see above to create 
 # see Plots Tutorial for parameter details
-example_scatter_plot = ScatterPlotData(
-    title=TITLE,
-    xaxis_title=X_TITLE,
-    yaxis_title=Y_TITLE,
-    xtrace=X_VALUES,
-    ytraces=Y_VALUES,
+converter.add_plot(
+    ScatterPlotData(
+        title="Something measured",
+        xaxis_title="Time (s) or something else",
+        yaxis_title="Some measurement (units)",
+        xtrace=X_VALUES,
+        ytraces={
+            "Agent 1" : Y_VALUES,
+        },
+    )
 )
-converter = TrajectoryConverter(input_data) # see above to create converter
-converter.add_plot(example_scatter_plot, "scatter")
 converter.write_JSON("output_file_name")
 ```
-___
 
-## Installation
-**Install Requires:** 
+### Render with meshes or PDB files
+If you'd like to render agents with arbitrary meshes instead of spheres, add a `DisplayData` for each agent type you'd like to be rendered with the mesh, and specify an OBJ mesh file. Alternately, you can provide a Protein Databank (PDB) file or PDB code to render atoms for a molecule. 
```

Comment:
1. "arbitrary meshes" seems like strange wording to me. Maybe "render agents as meshes"? They aren't arbitrary from the users point of view. 
2. I don't think we support just a PDB code? I could be wrong (@toloudis) but I think we still need a link or path to where that pdb file is. Or at least that's the only code I've looked at, maybe there is something that looks at the PDB first.

Reply:
1. yeah the reviewer used that term and I got stuck on it, will update
2. would be great if @toloudis can confirm one way or another and I can update

Reply:
Currently it has to be a full URL to a single .pdb file.

Reply:
in this case "arbitrary" means "any mesh of your choice" and not "a random mesh that is unrelated to anything".  It's arbitrary from the point of view of the software. :)

Reply:
yeah, I understand that, but it seems weird here since this is a doc that is supposed to be more user facing (I'm assuming).

Reply:
I agree with you.  I can see how users might interpret it that way.